longest_common_subsequence/LongestCommonSubsequence.java

```diff
@@ -0,0 +1,37 @@
+public class LongestCommonSubsequence {
+	//Function that returns Longest Common Subsequence of two strings
+	//Time Complexity - O( len(str1) * len(str2) ) 
+	//Space Complexity - O( len(str1)*len(str2) )
+	private static String longestCommonSubsequence(String str1, String str2) {
+		int [][] arr = new int[str1.length() + 1][str2.length() + 1];
+		for (int i = str1.length()-1; i >= 0; i--) {
```

Comment:
You must be able to see a **Resolve conflicts** button at the bottom. Go there and remove whatever is unwanted.